index.js

```diff
@@ -21,6 +30,65 @@ class GEmojiElement extends HTMLElement {
       this.appendChild(image)
     }
   }
+
+  static get observedAttributes(): Array<string> {
+    return ['tone']
+  }
+
+  attributeChangedCallback(name: string, oldValue: string, newValue: string) {
+    switch (name) {
+      case 'tone':
+        updateTone(this, parseInt(newValue, 10))
```

Comment:
We can avoid `parseInt` in multiple places by relying on the getter.

```suggestion
        updateTone(this)
```